MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -1043,6 +1061,14 @@ internal static void ApplyVisibilityUpdate(Actor actor, bool force = false)
 			}
 		}
 
+		private void CheckMaterialReferenceChanged(Guid id)
+		{
+			if (this != null && MaterialId == id && Renderer != null)
+			{
+				Renderer.sharedMaterial = (Material)App.AssetManager.GetById(id).Value.Asset;
```

Comment:
>.GetById(id).Value.Asse [](start = 56, length = 23)

Is this safe to just dereference when the GetBy Id can return null?

Reply:
This function is an event handler that is only called with valid asset IDs, so it will never return null as it's used in this PR.